app/controllers/leads_controller.rb

```diff
@@ -1,4 +1,74 @@
 class LeadsController < ApplicationController
   def index
+    @leads = []
+
+    if params[:q1].nil? == false
+
+      if params[:q1] == "" &&
+         params[:q2] == "" &&
```

Comment:
Use 4 (not 3) spaces for indenting a condition in an `if` statement spanning multiple lines.